README.md

```diff
@@ -12,19 +12,21 @@ See `./staging`
 - Tonomy login website: <https://tonomy-id-staging.tonomy.foundation>
 - Block explorer: <https://local.bloks.io/?nodeUrl=https%3A%2F%2Fblockchain-api-staging.tonomy.foundation&systemDomain=eosio>
 - Blockchain API: <https://blockchain-api-staging.tonomy.foundation>
-- Docs: <https://docs.tonomy.foundation>
+- Docs: <https://staging-docs.tonomy.foundation>
```

Comment:
see URLs here. Please make everything follow this

https://discord.com/channels/1029385699071381524/1039498420810952704/1118953569870160123

Reply:
still not correct